src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -78,12 +78,15 @@
     protected long maxBytes = Long.MAX_VALUE;
 
     private boolean shouldAttachSubscriberHandler;
+    private boolean shouldAttachUnsubscriberHandler;
```

Comment:
this doesn't make sense. The `shouldAttachSubscriberHandler` (which maybe we can rename to `shoudAttacheRecordsHandler` but not do that in this PR!) is there just to avoid or not attaching the handler on the Kafka consumer for getting messages automatically. This is the behavior we have with AMQP but for HTTP we avoid to attach the handler because the client asks for more messages sending more HTTP poll requests. On unsubscribe we don't need something like this.